canonical-websites/www.canonical.com
# Migrate taleo to greenhouse

## Done

* HR didn't want to use the greenhouse.io job listing page as the departments were incorrect, so using the api, we have replaced the older taleo page that will link to the hosted greenhouse job descriptions
* Robin added a new get_raw_json_feed template tag to read this format of json

## QA

- download this branch and `./run` the server
- look at http://0.0.0.0:8002/careers/all-vacancies and see that the page lists all open jobs, grouped by an external department field from greenhouse.io

## Issue / Card

Fixes #220 

## Screenshots

![image](https://user-images.githubusercontent.com/441217/33211815-2972c27e-d118-11e7-88bb-3da2b2eb9d60.png)
